providers/azure_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const Post = "POST"
```

Comment:
Instead of creating our own const, we could just use `http.MethodPost`

Reply:
Great idea.